core/templates/dev/head/pages/collection_editor/settings_tab/collection-permissions-card/collection-permissions-card.directive.ts

```diff
@@ -17,13 +17,13 @@
  * permissions.
  */
 
-oppia.directive('collectionPermissionsCard', [
+angular.module('collectionPermissionsCardModule').directive('collectionPermissionsCard', [
   'UrlInterpolationService', function(UrlInterpolationService) {
     return {
       restrict: 'E',
       templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
-        '/pages/collection_editor/settings_tab/' +
-        'collection_permissions_card_directive.html'),
+        '/pages/collection_editor/settings_tab/collection-permissions-card' +
```

Comment:
Missing `/` in end.

Reply:
Hello @amey-kudari Please do manual testing also to check if everything works fine or not.
Thanks